unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/EntityInformation/SmartItems/Parameters/SmartItemIntegerParameter.cs

```diff
@@ -0,0 +1,25 @@
+using DCL.Components;
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+public class SmartItemIntegerParameter : SmartItemUIParameterAdapter
+{
+    public TMP_InputField textParameterInputField;
+
+    const string parameterType = "integer";
```

Comment:
Can we use an `enum` for all the different parameter types?

Reply:
I proposed a Generic solution, but if it's not doable, this is the way to go!

Reply:
A refactor will be made to the whole part of the smart item in the next PR